README.md

```diff
@@ -8,6 +8,46 @@ This DOES NOT install a Chef client on your Jenkins server, that needs to be don
 The minimum Jenkins version we're building for is LTS 1.554.3.
 Get your bearings with <https://wiki.jenkins-ci.org/display/JENKINS/Plugin+tutorial>
 
+## Install plugin
+
+**Step 1**: Build .hpi file
+
+```
+mvn clean -DskipTest package
+```
+
+**Step 2**: In jenkins server
+
+- Go to **Manage Jenkins** -> **Manage Plugins** -> **Advanced** -> **Upload Plugin**
+
+- Select file chef-identity.hpi and upload to jenkins server
+
+## Chef Identity Setting
+
+Go to **Manage Jenkins** -> **Configure System** -> **Chef Identity Management**
+
+- Identity Name
+
+- user.pem key: Key pair download from AWS
+
+- knife.rb file: Default setting below
+
+```
+log_level                :info
+log_location             STDOUT
+cookbook_path            [File.expand_path('../../cookbooks' , __FILE__), File.expand_path('../../site-cookbooks' , __FILE__)]
+local_mode               true
```

Comment:
And let's make two `knife.rb` examples here.  One for this new zero/local mode and one for a non-local mode that would interact with a Chef Server.